src/actions/src.cr

```diff
@@ -1,19 +1,96 @@
+require "dppm/prefix/src"
+
 module DppmRestApi::Actions::Src
   extend self
   include RouteHelpers
 
+  struct MultiSet(K, V)
```

Comment:
Why creating a whole new type only used in `filters_and_return_keys`?

Reply:
Because it isn't an existing stdlib type, but this is the data structure which best encapsulates the data.

Reply:
Ok, but this means you have to add tests for it.
The logic implemented by it could be in the method, what advantage does it add?
One disadvantage is to have the logic in two places.

Reply:
Is it possible to group all filter related stuff inside a struct, which have both this logic, `filters_and_return_keys` and `filters`?